src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.wasm.cs

```diff
@@ -27,18 +27,13 @@ public partial class TextBox : Control
 
 		partial void OnFocusStateChangedPartial(FocusState focusState)
```

Comment:
So... does this mean that programmatic focus won't work? (Assuming it worked before)

Reply:
It may not no. Focus management on Wasm is not advanced enough.